common/flatpak-run.c

```diff
@@ -1669,12 +1669,24 @@ add_font_path_args (FlatpakBwrap *bwrap)
 static void
 add_icon_path_args (FlatpakBwrap *bwrap)
 {
+  g_autoptr(GFile) home = NULL;
+  g_autoptr(GFile) user_icons = NULL;
+
   if (g_file_test ("/usr/share/icons", G_FILE_TEST_IS_DIR))
     {
       flatpak_bwrap_add_args (bwrap,
                               "--ro-bind", "/usr/share/icons", "/run/host/share/icons",
                               NULL);
     }
+
+  home = g_file_new_for_path (g_get_home_dir ());
```

Comment:
Why not respect `XDG_DATA_HOME`?

Reply:
I was copying the code for user-fonts from a few lines up